source/modules/collection-api/sale/actions/create.md.erb

```diff
@@ -19,13 +19,13 @@ curl -X POST https://collection-prod.inspcdn.net/sale \
                 "event_id":           "23553",
                 "event_datetime_ids": ["101", "102"]
             }
-        ]
+        ],
         "payment": 
         {
             "method":              "credit_card",
             "installments":        1,
-            "first_six_digits_cc": 455326,
-            "last_four_digits_cc": 0012,
+            "first_six_digits_cc": "455326",
```

Comment:
Nit: Shouldn't this one be an int? In the http-collector it is an int

Reply:
Mmm I actually changed it to a String here https://github.com/inspetor/http-collection-service/pull/10/commits/13824b1d65d705c21f2412f76fecea9869f3f33d#diff-18e6f7fb7b09ee87e41e6ae05abb4b26R17 because JSON cannot interpret a number with leading zeroes as a `Number` type (and I thought it would be weird to have last four be a string, but first_six be an int)